# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2024/11
# add dummy parameters to build/build.py for cudnn_path and cuda_path, which are set by default by the jaxlib easyblock.
diff -ru jax-jax-v0.4.35/build/build.py jax-jax-v0.4.35_easyblockcompat/build/build.py
--- jax-jax-v0.4.35/build/build.py	2024-10-22 21:00:23.000000000 +0200
+++ jax-jax-v0.4.35_easyblockcompat/build/build.py	2024-11-19 12:35:46.524479324 +0100
@@ -549,6 +549,15 @@
       help_str="Same as update_requirements, but will consider dev, nightly "
                "and pre-release versions of packages.")
 
+  parser.add_argument(
+      "--cuda_path",
+      default="dummy",
+      help="compatibility with jaxlib.py easyblock")
+  parser.add_argument(
+      "--cudnn_path",
+      default="dummy",
+      help="compatibility with jaxlib.py easyblock")
+
   args = parser.parse_args()
 
   logging.basicConfig()
